ecosystem/sep-0006.md

```diff
@@ -540,6 +761,60 @@ The response should be a JSON object like:
       "fee_percent": 0
     }
   },
+  "deposit_indicative_quote": {
+    "USD": {
+      "enabled": true,
+      "authentication_required": true,
+      "fee_fixed": 5,
+      "fee_percent": 1,
+      "min_amount": 0.1,
+      "max_amount": 1000,
+      "fields": {
+        "email_address" : {
+          "description": "your email address for transaction status updates",
+          "optional": true
+        },
+        "amount" : {
+          "description": "amount in USD that you plan to deposit"
+        },
+        "country_code": {
+          "description": "The ISO 3166-1 alpha-3 code of the user's current address",
+          "choices": ["USA", "PRI"]
+        },
+        "type" : {
+          "description": "type of deposit to make",
+          "choices": ["SEPA", "SWIFT", "cash"]
+        }
+      }
+    }
+  },
+  "deposit_firm_quote": {
+    "USD": {
+      "enabled": true,
+      "authentication_required": true,
+      "fee_fixed": 5,
+      "fee_percent": 1,
+      "min_amount": 0.1,
+      "max_amount": 1000,
```

Comment:
Since the amount of the Stellar asset delivered to the user is determined when the request for quote is made, I don't think it makes sense to have the `min_amount` or `max_amount` properties here either.

Reply:
Removed those fields in https://github.com/stellar/stellar-protocol/pull/973/commits/ea0a84cd45149ffc6a757c6f96d25354e987802a.